src/components/Search.tsx

```diff
@@ -30,18 +31,37 @@ type SearchBarProps = {
   handleTextChanges: ( t: string ) => void,
 }
 
-const SearchBar = ( { handleTextChanges }: SearchBarProps ) => (
-  <View style={styles.searchBar}>
-    <Icon name="magnify" size={25} style={styles.searchIcon} />
-    <TextInput
-      placeholder="Koj"
-      style={styles.searchInputBox}
-      clearButtonMode="always"
-      autoCorrect={false}
-      autoCapitalize="none"
-      onChangeText={handleTextChanges}
-    />
-  </View>
-)
+const SearchBar = ( { handleTextChanges }: SearchBarProps ) => {
+  const { t } = useTranslation()
+
+  const changeLanguage = ( lng: string ) => {
+    i18n.changeLanguage( lng )
+  }
+  return (
+    <I18nextProvider i18n={i18n}>
+      <View style={styles.searchBar}>
+        <Icon name="magnify" size={25} style={styles.searchIcon} />
+        <TextInput
+          placeholder={t( 'searchBar.placeholder' )}
```

Comment:
I'd like to see these as referenced to something, not strings. 

I think moving the strings to the components will help.

Reply:
Can you give a concrete example / analogy here? I'm not sure I follow

Reply:
At best I was able to make this the "phrase" in english as the key. So that when you're reading the code you know what it's supposed to say in English. I do not know what more can be done, please let me know if you have any ideas.

Reply:
`t(strings.searchPlaceholder)`, that kinda thing